src/production/registration.rs

```diff
@@ -1,14 +1,15 @@
 //! Helpers for registering COM servers
 
+use crate::alloc::{format, string::String, vec::Vec};
```

Comment:
Are all these changes necessary if we know std is going to be used when this module is used?

Reply:
I'd like to finish removing the dependencies on `std`, even in the `registration` module. I just didn't want to do it all in one PR because updating this module required more than just changing paths. So I'd like to keep these imports.